Modules/Bridge/VTK/src/itkVTKImageExportBase.cxx

```diff
@@ -23,10 +23,7 @@ namespace itk
  * Constructor sets up information for the image-type indepenedent
  * callbacks implemented in this superclass.
  */
-VTKImageExportBase::VTKImageExportBase()
-{
-  m_LastPipelineMTime = 0;
-}
+VTKImageExportBase::VTKImageExportBase() { m_LastPipelineMTime = 0; }
```

Comment:
Honestly I like the original better, having the `m_LastPipelineMTime = 0;` on its own line. (No show stopper to me, just a matter of taste, I guess.)

Reply:
Please don't regress this PR in to a line by line critic of style. The goal is to update to a newer version clang format matching the current style as close as possible.

Reply:
@blowekamp Don't worry, I don't do 1000+ lines of code line by line. 😸 But it's good to know what kind of formatting changes we might expect from upgrading clang-format.

Reply:
I investigated options for minimizing code changes.  

I settled on changes that minimized differences with what clang-8 was doing:
AllowShortFunctionsOnASingleLine: All

I definitely like
 AllowShortFunctionsOnASingleLine: None 
much better, and that would make this line like it was. but other files would have changed.

I have a comment in the .clang-format file and plan to introduce the new option soon (this weekend :).

Hans